tests/unmanaged_models/models.py

```diff
@@ -142,3 +142,24 @@ class Meta:
 # table *will* be created (unless given a custom `through` as for C02 above).
 class Managed1(models.Model):
     mm = models.ManyToManyField(Unmanaged1)
+
+
+# Here is an unmanaged through model between two
+class UnmanagedIntermediate(models.Model):
+    d01 = models.ForeignKey('unmanaged_models.D01', models.CASCADE, db_column='d01_id')
+    d02 = models.ForeignKey('unmanaged_models,D02', models.CASCADE, db_column='d02_id')
```

Comment:
I was following the pattern in the previously defined models. I'll fix that up.

I don't know how that , didn't break things though!